src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,81 @@
+package duke.command;
+
+import duke.main.Ui;
+import duke.utils.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Delete command implementing command interface
+ */
+public class DeleteCommand implements Command {
+
+    /**
+     * attempt to delete task from task list and set ui response to the
+     * appropriate one
+     * @param task
```

Comment:
For Javadoc comments in our coding standard, it should start with a present tense that ends with an 's'. So it should be 'Attempts to delete....'.
Also, following the coding standard, it should have 1 blank line between the description and parameter section. It is also good to provide the description of the parameter, for example @param task The task of this execution.